pkgs/development/python-modules/mailman-rss/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, dateutil, future, requests, beautifulsoup4 }:
```

Comment:
I think we have `python-twitter` packaged so we should provide a `twitter` option.

Preferring the option name `twitter` because https://github.com/kyamagu/mailman-rss#install

Reply:
What do you mean? I kinda dislike packaging something I won't be using/testing, especially if I maintain it. Additionally, I'm not used to python things.

I think I understand that pypi packages are configurable, and that this package can be installed "with twitter functionality", right? How would this be done "the right way for pythong packages" in Nixpkgs?

Reply:
> I think I understand that pypi packages are configurable, and that this package can be installed "with twitter functionality", right? How would this be done "the right way for python packages" in Nixpkgs?

Pretty simply actually

```diff
diff --git a/pkgs/development/python-modules/mailman-rss/default.nix b/pkgs/development/python-modules/mailman-rss/default.nix
index 0271b50e405..30ebdd67007 100644
--- a/pkgs/development/python-modules/mailman-rss/default.nix
+++ b/pkgs/development/python-modules/mailman-rss/default.nix
@@ -1,5 +1,5 @@
 { stdenv, buildPythonApplication, fetchPypi
-, dateutil, future, requests, beautifulsoup4 }:
+, dateutil, future, requests, beautifulsoup4, withTwitter ? false, twitter  }:
 
 buildPythonApplication rec {
   pname = "mailman-rss";
@@ -10,7 +10,7 @@ buildPythonApplication rec {
     sha256 = "1brrik70jyagxa9l0cfmlxvqpilwj1q655bphxnvjxyganxf4c00";
   };
 
-  propagatedBuildInputs = [ dateutil future requests beautifulsoup4 ];
+  propagatedBuildInputs = [ dateutil future requests beautifulsoup4 ] ++ stdenv.lib.optional withTwitter twitter;
 
   doCheck = false;
 
```

Reply:
Right, pretty much as I thought (though wasn't sure of the expected name (`with$feature`).